### ISSUE TITLE
integrity check issue after upgrade from NC13->14

### ISSUE THREAD

**Issue by digitalkram** [PanCakeConnaisseur/user_backend_sql_raw#10]
hi there
after the ugrade to NC14 I have an issue with file integrity of user_backend_sql_raw

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- user_backend_sql_raw
	- INVALID_HASH
		- README.md
		- appinfo/info.xml

Raw output
==========
Array
(
    [user_backend_sql_raw] => Array
        (
            [INVALID_HASH] => Array
                (
                    [README.md] => Array
                        (
                            [expected] => c0d0c7ce72b05b1c5e82c36b3ab5ec346455ef6fff0c9e42a6bbd115401a2ce01b8758abb2d74b723233dc788823bfe44c31ca62a0b7608b1aef0e083298d668
                            [current] => ae7ca0fd9a7903127451df2d3f0210e6cf37c81930c00fb9e407ddac3a7714ee8ff0399491fb535273666e7d39704f95fd250e19c45383bd98894986885910eb
                        )

                    [appinfo/info.xml] => Array
                        (
                            [expected] => 6ea2839992a63b58d262ddeed9ae663146e07471882b1f336e3fd2c3c259e28013f583a10f9079f8470bdcff09219559916230dd48b6bf6c0cf7e5c4882df8a9
                            [current] => 187bd6f1ae6ab605bdf2bc9f4ef46d424a23893ba4ffd5ec45af7db41ad8947cce32cb99ebbdd9310ce40fcb82f0af10bc3dcbd3015231829b08b3783efe7ae6
                        )

                )

        )

)
```

I read the documentation about it and thought I just might have to replace those files with the proper version. However when I redownload the latest release and check the sha512sum checksum of the two files in question I again have the "current" and not the "expected" one.

So either the checksums in ./appinfo/signature.json are wrong or two dated files are distributed...?

Or I totally miss the point.

Thanks for any input provided!

Cheers

**Comment by ChiefPete** [comment]
I'm see the same INVALID_HASH as well as EXTRA_FILE LICENSE.

I have version 1.0.3 installed on Nextcloud version 14.0.4

Thanks.

**Comment by PanCakeConnaisseur** [comment]
This is fixed in v1.0.4 which was releases just now.

**Comment by PanCakeConnaisseur** [comment]
If the error message persists, you might have to click on `Rescan`.
